http/examples/get/client_async.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+import aiohttp
```

Comment:
This was an experiment to compare the throughput when doing multiple GET requests in parallel. But this example uses much more memory than I expected and runs very slowly. Anyone know what I'm doing wrong here?

Reply:
I think this is fetching the HTTP responses (potentially) in parallel - the part under `asyncio.run` -, but then decodes those HTTP responses serially - the regular `for` loop invoking `open_stream` on each response in turn.

We would need to wire `StreamDecoder` into Python and provide a higher-level async IPC read API around it. This is probably off-topic for this.

For now, you could just make the blocking client `client.py` parallel by using [ThreadPoolExecutor](https://docs.python.org/3/library/concurrent.futures.html#concurrent.futures.ThreadPoolExecutor).

Reply:
Thanks — for now I'm just going to remove this async Python example because I don't think it adds value here.